From c61b8015a97b7d91f0c3c9e840f2c96ce5cf9730 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Thu, 16 May 2019 10:50:25 +0700
Subject: [PATCH 046/458] arm64: dts: renesas: hihope-common: enable
 TMU{0,1,2,3,4}

This patch enables TMU{0,1,2,3,4} support for both Hihope-rzg2m and
Hihope-rzg2n platform.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index 40dc9f0..fd765d8 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -447,3 +447,23 @@
 &cmt1 {
 	status = "okay";
 };
+
+&tmu0 {
+	status = "okay";
+};
+
+&tmu1 {
+	status = "okay";
+};
+
+&tmu2 {
+	status = "okay";
+};
+
+&tmu3 {
+	status = "okay";
+};
+
+&tmu4 {
+	status = "okay";
+};
-- 
2.7.4

